page-index/page-index.js

```diff
@@ -491,6 +491,11 @@ const pageIndex = [
                 type: PAGE,
                 href: "prometheus-adapter",
                 children: []
+            }, {
+                Title: "{insert django tutorial title here}",
+                type: PAGE,
+                href: "tutorial-howto-monitor-django-prometheus",
+                children: []
```

Comment:
You were missing this addition to the `page-index.js` file.  The page can't be displayed without it.  Please modify this to add the title you want to appear on the sidebar (keep in mind the length)